assets/js/base/context/checkout-context.js

```diff
@@ -55,7 +58,7 @@ const CheckoutProvider = ( {
 		checkoutHasError,
 		activePaymentMethod,
 		placeOrderLabel,
-		notices,
+		placeOrderAction.notices,
```

Comment:
Will this be undefined? (Since `placeOrderAction` is an empty placeholder function above)

I didn't see any JS errors, so not sure if this code is firing.

Reply:
Will be fixed after master rebase.